jobs/jobs/categories.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from typing import List, Tuple
+
+import jobs.models as models
+import jobs.schemas as schemas
+import jobs.utils as utils
+
+logger = logging.getLogger(__name__)
+
+
+async def prepare_for_update(
+    current_job: models.CombinedJob,
+    new_job: schemas.JobParamsToChange,
+    current_tenant: str,
+    jw_token: str,
+) -> Tuple[List[str], List[schemas.CategoryLinkInput]]:
+    logger.info("Prepare categories for update")
```

Comment:
Do we need this log? We log later certain categories for certain actions

Reply:
This log message could be removed. However, if something inside the function raises an exception, the last message in the log stream, 'Prepare categories for update,' can easily help us identify the last called function. 

I agree that we could find the complete list of calls in the stack trace, but this message serves as an additional logging point.